onnx/numpy_helper.py

```diff
@@ -36,10 +36,9 @@ def to_array(tensor: TensorProto, base_dir: str = "") -> np.ndarray:
         raise TypeError("The element type in the input tensor is not defined.")
 
     tensor_dtype = tensor.data_type
-    np_dtype = mapping.TENSOR_TYPE_TO_NP_TYPE[tensor_dtype]
-    storage_type = mapping.TENSOR_TYPE_TO_STORAGE_TENSOR_TYPE[tensor_dtype]
```

Comment:
In PR #4510, I remove the use of dictionary `TENSOR_TYPE_TO_STORAGE_TENSOR_TYPE`. It think use int8 or uint8 to store bool should not be allowed.